### DISCUSSION TITLE
Janet 1.36.0

### DISCUSSION THREAD

**Discussion by bakpakin** [janet-lang/janet#1499] [General]
## What's Changed
* Add support for the AAPCS64 calling convention by @SyrupThinker in https://github.com/janet-lang/janet/pull/1468
* Add ctrl + left/right arrow support to the REPL by @Darazaki in https://github.com/janet-lang/janet/pull/1471
* Add default output path for --compile (-c) by @Andriamanitra in https://github.com/janet-lang/janet/pull/1472
* Do not warn when loop body is empty by @sogaiu in https://github.com/janet-lang/janet/pull/1476
* Fix fallback support in polymorphic compare by @pyrmont in https://github.com/janet-lang/janet/pull/1480
* Prefix bin+sep to add-bin dest argument by @sogaiu in https://github.com/janet-lang/janet/pull/1485
* Move bundle/add-bin changelog line to unreleased by @sogaiu in https://github.com/janet-lang/janet/pull/1492
* Move other unreleased change out of v1.35.2 section by @pyrmont in https://github.com/janet-lang/janet/pull/1494
* Don't try to copy janet.def by @sogaiu in https://github.com/janet-lang/janet/pull/1498

## New Contributors
* @SyrupThinker made their first contribution in https://github.com/janet-lang/janet/pull/1468
* @Darazaki made their first contribution in https://github.com/janet-lang/janet/pull/1471

**Full Changelog**: https://github.com/janet-lang/janet/compare/v1.35.2...v1.36.0

<hr /><em>This discussion was created from the release <a href='https://github.com/janet-lang/janet/releases/tag/v1.36.0'>Janet 1.36.0</a>.</em>

**Comment by jdf-id-au** [comment]
`janet-v1.36.0-macos-x64.tar.gz` seems to be built for arm64, not x64

```
$ tar xfvz janet-v1.36.0-macos-x64.tar.gz
...
$ dyld_info janet-v1.36.0-macos/bin/janet
janet-v1.36.0-macos/bin/janet [arm64]
...
```

**Reply by bakpakin** [reply]
Woops, thanks for pointing that out. Seems as though github decided that `macos-latest` will run on aarch64 machines now by default, which I guess makes sense. I will rename the current upload to `-aarch64`, but we will need to update CI to build for macos + intel now.

**Reply by jdf-id-au** [reply]
But both `make && make install` and `brew install janet --HEAD` worked flawlessly, thank you.
